plugins/net_plugin/net_plugin.cpp

```diff
@@ -4421,29 +4408,17 @@ namespace eosio {
    }
 
    void net_plugin::plugin_startup() {
-      try {
-         my->plugin_startup();
-      } catch( ... ) {
-         // always want plugin_shutdown even on exception
-         plugin_shutdown();
-         throw;
-      }
+      my->plugin_startup();
    }
-      
 
    void net_plugin::handle_sighup() {
       fc::logger::update( logger_name, logger );
    }
 
    void net_plugin::plugin_shutdown() {
-      try {
-         fc_ilog( logger, "shutdown.." );
-
-         my->plugin_shutdown();
-         app().executor().post( 0, [me = my](){} ); // keep my pointer alive until queue is drained
-         fc_ilog( logger, "exit shutdown" );
-      }
-      FC_CAPTURE_AND_RETHROW()
+      fc_dlog( logger, "shutdown.." );
+      my->plugin_shutdown();
+      fc_ilog( logger, "exit shutdown" );
```

Comment:
`net_plugin` is redundant.
```
info  2024-10-16T12:12:03.410 nodeos    net_plugin.cpp:4421           plugin_shutdown      ] exit shutdown
```

Reply:
depends how you look at the log, zigzag fashion or more linearly, but OK.

Reply:
tbh I'm not really sure how useful these shutdown logs are; I almost suggested downgrading them to debug

Reply:
I think they only existed because our shutdown logic was so suspect and we wanted these hints of where on shutdown things may have gone wrong. With this PR, changing them all to debug I think makes sense.